src/nunit.analyzers/ClassicModelAssertUsage/AreEqualClassicModelAssertUsageCodeFix.cs

```diff
@@ -50,6 +59,16 @@ protected override void UpdateArguments(Diagnostic diagnostic, List<ArgumentSynt
             // Then we have to remove the 1st argument because that's now in the "Is.EqualTo()"
             arguments.RemoveAt(0);
```

Comment:
The `Insert(2)` might insert at the wrong spot.
This `RemoveAt` might remove the actual parameter instead of the expected one.